Bindings/bindings.json

```diff
@@ -455,6 +455,20 @@
                     ]
                 },
                 {
+                    "name": "subscriptionName",
+                    "value": "string",
+                    "defaultValue": "mysubscription",
+                    "required": true,
+                    "label": "Subscription name",
+                    "help": "This is the name of the subscription from which the message will be read.",
```

Comment:
Recommend using the same or similar wording that the ServiceBusTriggerAttribute itself uses for these parameters